backend/internal/domain/helpers/collections.go

```diff
@@ -0,0 +1,10 @@
+package helpers
```

Comment:
Best practice is to avoid utils/helpers/...
The package name serves for readability also, and helpers does not add any context. It is better to have a small amount of duplication, than a wrong abstraction.
If you need helpers, you should (in this case) provide a `slices` package, and then the call would be:
```go
slices.Contains(s, element)
```
You can even go a step further. You are currently allowing string | int. Why not `comparable` to make it general purpose?

Reply:
Removed helpers folder